src/area-chart/model/use-chart-model.ts

```diff
@@ -273,10 +274,10 @@ export default function useChartModel<T extends AreaChartProps.DataTypes>({
     };
 
     // A callback for svg blur to clear all highlights unless the popover is pinned.
-    const onSVGBlur = () => {
+    const onSVGBlur = (event: React.FocusEvent<Element>) => {
       // Pinned popover stays pinned even if the focus is lost.
       // If blur is not caused by the popover, forget the previously highlighted point.
-      if (!interactions.get().isPopoverPinned) {
+      if (!nodeBelongs(containerRef.current, event.relatedTarget) && !interactions.get().isPopoverPinned) {
```

Comment:
This fixes not being able to navigate into the popover. In that situation we blur the SVG but not the popover, so e check if we blur the container which contains both the SVG and the popover. This makes it equivalent to the behavior of the mixed line-bar chart [here](https://github.com/cloudscape-design/components/blob/main/src/mixed-line-bar-chart/chart-container.tsx#L393) and the pie chart [here](https://github.com/cloudscape-design/components/blob/main/src/pie-chart/pie-chart.tsx#L263).